table/tables/mutation_checker_test.go

```diff
@@ -156,7 +164,144 @@ func TestCheckRowInsertionConsistency(t *testing.T) {
 	}
 
 	for caseID, data := range testData {
-		err := checkRowInsertionConsistency(sessVars, data.rowToInsert, data.rowInsertion, data.columnIDToInfo, data.columnIDToFieldType)
+		err := checkRowInsertionConsistency(
+			sessVars, data.rowToInsert, data.rowInsertion, data.columnIDToInfo, data.columnIDToFieldType,
+		)
 		require.Equal(t, data.correct, err == nil, "case id = %v", caseID)
 	}
 }
+
+func TestCheckIndexKeys(t *testing.T) {
+	/*
+		dimensions of the domain of checkIndexKeys:
+		1. location										*2
+		2. table structure
+			(1) unique index/non-unique index			*2
+			(2) clustered index							*2
+			(3) string collation						*2
+			We don't test primary clustered index and int handle, since they should not have index mutations.
+	*/
```

Comment:
Consider using `//` instead of `/* */` and use spaces to align the content in the comment?